### PR TITLE
Fix some instances of \*\* in markdown

### PR DESCRIPTION
**Pull request by cw118**
#### Summary/Motivation
- Fixes/removes a few occurences of `\*\*` in Markdown files
- Replaces "invisible U+00a0" characters with regular spaces
- Small formatting fixes (mostly fixing double/extra spaces)

I had some difficulty "guessing" what the intended formatting was where there were escaped asterisks (`\*`), if something is still incorrect I'll be happy to fix it :)

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/server-side/express_nodejs/forms/delete_author_form/index.md [hunk 1]
```diff
@@ -38,15 +38,15 @@ exports.author_delete_get = function(req, res, next) {
 };
 ```
 
-The controller gets the id of the `Author` instance to be deleted from the URL parameter (`req.params.id`). It uses the `async.parallel()` method to get the author record and all associated books in parallel. When both operations have completed it renders the **`author_delete`\*\***.pug\*\* view, passing variables for the `title`, `author`, and `author_books`.
+The controller gets the id of the `Author` instance to be deleted from the URL parameter (`req.params.id`). It uses the `async.parallel()` method to get the author record and all associated books in parallel. When both operations have completed it renders the **`author_delete`.pug** view, passing variables for the `title`, `author`, and `author_books`.
```
Comment (hamishwillee) [inline files/en-us/learn/server-side/express_nodejs/forms/delete_author_form/index.md:41]:
```suggestion
The controller gets the id of the `Author` instance to be deleted from the URL parameter (`req.params.id`). It uses the `async.parallel()` method to get the author record and all associated books in parallel. When both operations have completed it renders the **author_delete.pug** view, passing variables for the `title`, `author`, and `author_books`.
```

File: files/en-us/learn/server-side/express_nodejs/forms/delete_author_form/index.md [hunk 2]
```diff
@@ -38,15 +38,15 @@ exports.author_delete_get = function(req, res, next) {
...
-> **Note:** If  `findById()` returns no results the author is not in the database. In this case there is nothing to delete, so we immediately render the list of all authors.
+> **Note:** If `findById()` returns no results the author is not in the database. In this case there is nothing to delete, so we immediately render the list of all authors.
 >
 > ```js
-> }, function(err, results) {
+>   function(err, results) {
 >     if (err) { return next(err); }
 >     if (results.author==null) { // No results.
->         res.redirect('/catalog/authors')
+>       res.redirect('/catalog/authors');
```
Comment (hamishwillee) [inline files/en-us/learn/server-side/express_nodejs/forms/delete_author_form/index.md:49]:
```suggestion
>         res.redirect('/catalog/authors');
```

File: files/en-us/mozilla/add-ons/webextensions/chrome_incompatibilities/index.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ The rest of this page summarizes these and other incompatibilities.
 
 ## JavaScript APIs
 
-### \*chrome.\** and *browser.\*\* namespace
+### *chrome.\** and *browser.*\* namespace
```
Comment (hamishwillee) [inline files/en-us/mozilla/add-ons/webextensions/chrome_incompatibilities/index.md:31]: We could strip these altogether but I think this is fine. Minor mode to make it clear intent. 
```suggestion
### _chrome.\*_ and _browser.\*_ namespace
```

File: files/en-us/mozilla/add-ons/webextensions/match_patterns/index.md [hunk 4]
```diff
@@ -93,7 +93,7 @@ The value for the _path_ matches against the string which is the URL path plus t
 
 Neither the [URL fragment identifier](https://en.wikipedia.org/wiki/Fragment_identifier), nor the `#` which precedes it, are considered as part of the _path_.
 
-> **Note:** The path pattern string should not include a port number. Adding a port, as in: _"http\://localhost:1234/\*"_ causes the match pattern to be ignored. However, "_http\://localhost:1234_" will match with "\*http\://localhost/\*\*"
+> **Note:** The path pattern string should not include a port number. Adding a port, as in: _"http\://localhost:1234/\*"_ causes the match pattern to be ignored. However, _"http\://localhost:1234"_ will match with _"http\://localhost/\*"_.
```
Comment (hamishwillee) [inline files/en-us/mozilla/add-ons/webextensions/match_patterns/index.md:96]: Just putting these in code style probably "safer"
```suggestion
> **Note:** The path pattern string should not include a port number. Adding a port, as in: `http://localhost:1234/*` causes the match pattern to be ignored. However, `http://localhost:1234` will match with `http://localhost/*`.
```

File: files/en-us/mozilla/add-ons/webextensions/chrome_incompatibilities/index.md (hunk 3 above)
Comment (cw118) [reply files/en-us/mozilla/add-ons/webextensions/chrome_incompatibilities/index.md]: Thank you! I'm planning on adding the suggestions to a batch in order to make all changes in a single commit.

**Review by hamishwillee — APPROVED** [review]
Thanks @cw118 - looks good.

I've made (accepted) a couple of tiny changes and will merge shortly.

**Comment by cw118** [comment]
@hamishwillee Ah thank you, was just about to commit the suggestions myself!

**Pull request merged**